nservicebus/sqlserver/index.md

```diff
@@ -28,38 +28,63 @@ The SQL Server transport is a hybrid queueing system which is neither store-and-
  * No additional licensing and training costs, as majority of Microsoft stack organizations already have SQL Server installed and have knowledge required to run it
  * Great tooling (SSMS)
  * Maximum throughput for any given endpoint is on par with MSMQ
+ * Free to start (Express edition)
  * Queues support competing consumers (multiple instances of same endpoint feeding off of same queue) so scale-out doesn't require using [distributor](/nservicebus/scalability-and-ha/distributor/)
 
 
 ### Disadvantages
 
  * No local store-and-forward mechanism, meaning that when SQL Server instance is down the endpoint cannot send nor receive messages
- * In centralized deployment maximum throughput applies for the whole system, not individual endpoints. If SQL Server on a given hardware can handle 2000 msg/s, each one of the 10 endpoints deployed on this machine can only receive 200 msg/s (on average).
+ * In centralized deployment maximum throughput applies for the whole system, not individual endpoints. If SQL Server on a given hardware can handle 2000 msg/s, each one of the 10 endpoints deployed on this machine can only receive 200 msg/s (on average)
+ * Queue tables are polled periodically to check if receive operation should be performed. This may lead to delays in message delivery for endpoints that experience low rate of incoming messages.
```

Comment:
> This may lead to delays in message delivery for endpoints that experience low rate of incoming messages.

Isn't the poll period constant? Or we try to be smart about this?  If the latter then I'd add something like "because the polling period is determined dynamically/optimized for the average rate". Can the user control it, at least the default? If so then also worth mentioning.